### PR TITLE
Pnt 17 conversion from fct addresses to peg net asset addresses fixed

### PR DESCRIPTION
**Pull request by PaulSnow**
This also includes the travis setup

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# Get gox cross-compilation library
+# Move into pegnetMining folder for build
+before_install:
+  - go get github.com/mitchellh/gox
+  - cd pegnetMining
+
+
+# Get specific factom branch
+script: 
+  - go get github.com/FactomProject/factom@v1-rollup
```
Comment (stackdump) [inline .travis.yml]: can this be locked using go.mod?

Comment (MitchellBerry) [reply .travis.yml]: quite sure go.mod being included now makes this call redundant
`go get ...@v1-rollup` creates a go.mod if there isn't one already

Comment (Emyrk) [reply .travis.yml]: It should already be locked in the go.mod

Comment (Emyrk) [reply .travis.yml]: I think this was done before go mod was in place

File: support/uitls.go [hunk 2]
```diff
@@ -220,12 +212,39 @@ func ConvertECAddressToUser(addr []byte) string {
 	return base58.Encode(userd)
 }
 
-
 // Convert a User facing Factoid or Entry Credit address
 // or their Private Key representations
 // to the regular form.  Note validation must be done
 // separately!
-func ConvertFctECUserStrToAddress(userFAddr string) string {
+func ConvertUserStrFctEcToAddress(userFAddr string) (string, error) {
```
Comment (Emyrk) [inline support/uitls.go]: The factom lib has this, and it checks the checksum too:
https://github.com/FactomProject/factom/blob/master/addresses.go#L43

File: support/utils_test.go [hunk 3]
```diff
@@ -7,19 +7,77 @@ import (
 	"encoding/hex"
 	"errors"
 	"fmt"
+	"github.com/FactomProject/factoid"
 	. "github.com/pegnet/pegnet/support"
 	"testing"
 )
 
-func TestBurn(t *testing.T) {
+func TestConvertFctToPegAssets(t *testing.T) {
```
Comment (Emyrk) [inline support/utils_test.go]: Should probably include hard-coded test vectors to verify addresses do not change in the future.

FCT -> PNT addr list

**Pull request merged**